### PR TITLE
Prac 05 feedback

### PR DESCRIPTION
**Pull request by SallySheldon**
Please review my work for Prac 05 @HenevianAcero - I have diverged from supplied solutions for some code so would appreciate your comments especially on those aspects.

### CODE REVIEW

File: prac_05/hex_colours.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+print(COLOUR_TO_CODE)
+
+colour_name = input("\nEnter colour name: ").lower()
+while colour_name != "":
+    try:
+        print(f"{colour_name.title()} is colour code {COLOUR_TO_CODE[colour_name]}")
+    except KeyError:
+        print("Sorry, that colour name isn't in our list.")
+    colour_name = input("\nEnter colour name: ").lower()
```
Comment (HenevianAcero) [inline prac_05/hex_colours.py:23]: Good use of the EAFP approach. It's also good that you added '\n' to ensure that a new line is created when asking the user for the colour name. However, is it still necessary to add if you know that the input command will automatically create a new line?

File: prac_05/state_names.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    # LBYL approach:
+    # if state_code in CODE_TO_NAME:
+    #     print(state_code, "is", CODE_TO_NAME[state_code])
+    # else:
+    #     print("Invalid short state")
+
+    # EAFP approach:
+    try:
+        print(state_code, "is", CODE_TO_NAME[state_code])
+    except KeyError:
```
Comment (HenevianAcero) [inline prac_05/state_names.py:22]: Good use of the EAFP approach.

File: prac_05/hex_colours.py (hunk 1 above)
Comment (SallySheldon) [reply prac_05/hex_colours.py:23]: Thanks @HenevianAcero, I understand what you mean. I've stuck with the \n character for now as I found that the spacing helps the program user see more clearly where the prompt is and what's going on.

File: prac_05/state_names.py (hunk 2 above)
Comment (SallySheldon) [reply prac_05/state_names.py:22]: Thanks @HenevianAcero

**Pull request merged**